runelite-client/src/main/java/net/runelite/client/plugins/fightcave/FightCavePlugin.java

```diff
@@ -53,6 +56,74 @@
 
 	private JadAttack attack;
 
+	private static final ArrayList<Integer> FIGHT_CAVE_REGIONS = new ArrayList<Integer>(Arrays.asList(9294, 9295, 9296, 9550, 9551, 9552, 9806, 9807, 9808));
+
+	private static final int[][] WAVE_ENEMIES = {//I could have just written a function, but this is faster to write :shrug:
+			{22},
```

Comment:
I think a lookup table like this is fine as is.
I recommend you add a comment before this saying each wave is a list of monsters that are present in that wave. That would be a more helpful comment.